test/metabase/api/pivots.clj

```diff
@@ -12,38 +13,41 @@
 (defn pivot-query
   "A basic pivot table query"
   []
-  (-> (mt/mbql-query orders
-        {:aggregation [[:count] [:sum $orders.quantity]]
-         :breakout    [[:fk-> $orders.user_id $people.state]
-                       [:fk-> $orders.user_id $people.source]
-                       [:fk-> $orders.product_id $products.category]]})
-      (assoc :pivot_rows [1 0]
-             :pivot_cols [2])))
+  (mt/dataset sample-dataset
+    (-> (mt/mbql-query orders
+          {:aggregation [[:count] [:sum $orders.quantity]]
+           :breakout    [$orders.user_id->people.state
+                         $orders.user_id->people.source
+                         $orders.product_id->products.category]})
+        (assoc :pivot_rows [1 0]
+               :pivot_cols [2]))))
```

Comment:
I reworked a bunch of these tests to use the more conscise `$ids` shorthand for `:fk->` forms

Reply:
I can't help but think we have too many different ways to do this.

Reply:
it's all really just shorthand for the same thing... it makes it a lot more convenient to write queries from the REPL IMO or to write tests. I would rather write `$user_id->people.state` than `[:fk-> [:field-id (mt/id :orders :user_id)] [:field-id (mt/id :people :state)]]`. I think it's more readable this way too.

Added benefit: if we implement something like #14057 and replace `:field-id` and `:fk->` with a new unified `:field` clause or something like that we can tweak these macros to expand to the new replacement forms.